main.js

```diff
@@ -3,10 +3,32 @@ var reservations = {
   'Ted': { claimed: true }
 }
 
-var name = prompt('Please enter the name for your reservation');
-
-var claimReservation = function () {
-  // write your code here!
+let namesArray=Object.keys(reservations)
+let c=0;
+var claimReservation= function(input){
+let name= document.getElementById("input").value;
+  for(let i of namesArray){
```

Comment:
this whole for-loop is extraneous
one of the nice things about objects is that you can figure out right away whether something exists in the object or not:

```
reservations["Ted"] // resolves to true because "Ted" is a key of reservations
reservations["Shubert"] // resolves to false because "Shubert" is not a key of reservations
```